tensorflow_serving/model_servers/main.cc

```diff
@@ -128,15 +129,24 @@ tensorflow::Status LoadCustomModelConfig(
 }
 
 ModelServerConfig BuildSingleModelConfig(const string& model_name,
-                                         const string& model_base_path) {
+                                         const string& model_base_path,
+                                         const string& model_version_policy) {
```

Comment:
Please make this the enum type, not a string type, and move the parsing outside of this function.